### PR TITLE
feat: add `unocss` example

### PR DESCRIPTION
**Pull request by nexxeln**
This PR adds an [UnoCSS](https://github.com/unocss/unocss) example.

Screenshot

### CODE REVIEW

File: unocss/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "npm run build:css && remix build",
+    "build:css": "unocss './app/**/*.{js,ts,jsx,tsx}' -o ./app/styles/uno.css",
+    "dev": "concurrently \"npm run dev:css\" \"remix dev\"",
```
Comment (MichaelDeBoey) [inline unocss/package.json:7]: Can we please use `npm-run-all` instead, like we do in our other examples & our templates?

https://github.com/remix-run/examples/blob/c149358fe3b1f7fd465c763209488ce67d0a6969/tailwindcss/package.json#L4-L13

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: unocss/package.json (hunk 1 above)
Comment (nexxeln) [reply unocss/package.json:7]: Sorry, didn't notice that. Will push a fix soon.

**Review by mcansh — APPROVED** [review]

**Review by machour — APPROVED** [review]

File: unocss/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "sideEffects": false,
+  "scripts": {
+    "build": "run-s \"build:*\"",
+    "build:remix": "remix build",
+    "build:css": "unocss './app/**/*.{js,ts,jsx,tsx}' -o ./app/styles/uno.css",
+    "dev": "run-p \"dev:*\"",
+    "dev:remix": "remix dev",
+    "dev:css": "unocss './app/**/*.{js,ts,jsx,tsx}' -o ./app/styles/uno.css --watch",
+    "start": "remix-serve build"
+  },
```
Comment (MichaelDeBoey) [inline unocss/package.json:12]: What about having a `generate:css` script as well, like we have in the `tailwindcss` example?

```suggestion
  "scripts": {
    "build": "run-s \"build:*\"", 
    "build:css": "npm run generate:css", 
    "build:remix": "remix build", 
    "dev": "run-p \"dev:*\"", 
    "dev:css": "npm run generate:css -- --watch", 
    "dev:remix": "remix dev", 
    "generate:css": "unocss './app/**/*.{js,ts,jsx,tsx}' -o ./app/styles/uno.css", 
    "start": "remix-serve build"
  },
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: unocss/package.json (hunk 2 above)
Comment (nexxeln) [reply unocss/package.json:12]: It's just adding even more complexity in my opinion. But sure I'll add it.

Comment (nexxeln) [reply unocss/package.json]: Done

**Review by MichaelDeBoey — APPROVED** [review]

**Pull request merged**